openstack/nova/templates/hypervisors/_kvm-deployment.yaml.tpl

```diff
@@ -92,8 +99,12 @@ spec:
               readOnly: true
             - mountPath: /nova-patches
               name: nova-patches
+            - mountPath: /etc/nova/rootwrap.conf
```

Comment:
Doesn't nova need any `/etc/rootwrap.d/` stuff?

Reply:
kvm nodes need it because they execute alot of local cli commands of libvirt/etc.

Reply:
But `rootwrap.conf` is not enough most of the time. The real config is in files in  `/etc/rootwrap.d/`. We just recently added one for cinder.

Reply:
Indeed, nova also seems to have introduced new rootwrap.d files:
https://github.com/sapcc/nova/tree/stable/queens-m3/etc/nova/rootwrap.d

Reply:
nova will use the preinstalled rootwrap.d files

Reply:
They get installed automatically? They were'nt for cinder.

Reply:
https://github.com/sapcc/helm-charts/blob/master/openstack/nova/templates/etc/_rootwrap.conf.tpl#L2

Reply:
yes nova setup.py installs them